src/mocks/handlers.ts

```diff
@@ -1,16 +1,16 @@
 import { http, HttpResponse } from 'msw'
 // import booksFixture from '../mocks/fixtures/booksFixture.json'
-import reviewsFixture from '../mocks/fixtures/reviewsFixture.json'
+// import reviewsFixture from '../mocks/fixtures/reviewsFixture.json'
 
 export const handlers = [
   // http.get('http://localhost:8080/books', () => {
   //   // ...and respond to them using this JSON response.
   //   return HttpResponse.json(booksFixture)
   // }),
 
-  http.get('*/reviews*', () => {
-    return HttpResponse.json(reviewsFixture)
-  }),
+  // http.get('*/reviews*', () => {
+  //   return HttpResponse.json(reviewsFixture)
```

Comment:
When you want to see actual response instead of the handler you can use [passthrough](https://mswjs.io/docs/api/passthrough/), it's cleaner than commenting out the hanlders